src/agentic/db/db_manager.py

```diff
@@ -0,0 +1,134 @@
+from datetime import datetime, UTC
+from typing import Dict, Optional
+from sqlmodel import Session, SQLModel, create_engine, select
+from pathlib import Path
+from copy import deepcopy
+
+from agentic.db.models import Run, RunLog
+from agentic.events import FinishCompletion
+
+# Database setup and management
+class DatabaseManager:
+    def __init__(self, db_path: str = "~/.agentic/database.db"):
```

Comment:
Maybe a less generic name? 'agent_runs.db' or something

FWIW the "agentic list-secrets" was sort of intended to be "global" in the way that you might set Env vars in your bash profile, which is why I put it in ~/agentic. Kinda feels to me like this data is specific to your "agentic" installation - like you might install agentic and run it from multiple projects. So I would have imagined this would end up in the `runtime` directory which we make the default dir for agents.